### ISSUE TITLE
Fix/Remove README in incubator

### ISSUE THREAD

**Issue by Nuru** [cloudposse/charts#199]
The README at https://github.com/cloudposse/charts/blob/0.6.0/incubator/README.md refers to the opsgoodness chart repo at http://charts.opsgoodness.com but that repo does not seem to be working. In fact, it's not clear that this README is actually meant for that directory. It should be fixed if not deleted.

**Comment by osterman** [comment]
Yes, I think this was accidentally committed long ago. We should just delete it.
